### PR TITLE
Update dependency and add missing istanbul dependency

### PR DESCRIPTION
**Pull request by mohsen1**
Also fix a test that was breaking due to swagger-tools change and clean up travis.yaml file

### CODE REVIEW

**Comment by mohsen1** [comment]
@IvanGoncharov please review

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 sudo: false
 language: node_js
 script:
-  - npm test && npm run coveralls
+  - npm test
+  - npm run coveralls
```
Comment (IvanGoncharov) [inline .travis.yml]: From Travis docs:
> When one of the build commands returns a non-zero exit code, the Travis CI build runs the subsequent commands as well, and accumulates the build result.

http://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step

So if `npm test` will fail Travis will try to execute `npm run coveralls` and it's produce more errors which will make error diagnostic harder.

File: test/output/fixable.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "basePath": "/v1",
+    "basePath": "/v1/",
```
Comment (IvanGoncharov) [inline test/output/fixable.json]: Did you fix `npm test`?

Comment (mohsen1) [inline test/output/fixable.json]: that was a lazy "fix". Let me revert back and see what's going on. I'm running `iojs` locally and had to make this change to make test pass. But now in travis, only `iojs` is passing!

Comment (IvanGoncharov) [inline test/output/fixable.json]: For some reason result different on `node.js` and `iojs`.

File: .travis.yml (hunk 1 above)
Comment (mohsen1) [inline .travis.yml]: That's okay.

File: test/output/fixable.json (hunk 2 above)
Comment (mohsen1) [inline test/output/fixable.json]: Well, it's an `iojs` specific issue. I'm guessing it has to do something with `url` module and breaking changes in `iojs`

Comment (IvanGoncharov) [inline test/output/fixable.json]: Can you split `iojs` into separate PR.
So it will not block release?

Comment (mohsen1) [inline test/output/fixable.json]: Lets figure iojs issue. It'a outputting different basePath which is serious. A lot of people (including me) are using iojs.

**Comment by mohsen1** [comment]
I just confirmed that it's iojs' new URL module breaking changes that is causing the failure. I'm digging in to see if it's iojs' bug or not.

**Comment by IvanGoncharov** [comment]
Ok. I'm currently working on feature that will extend API, we discussed it on Gitter.
I think it better to push it in scope of `1.0.0`.
I will submit PR tomorrow.

**Comment by mohsen1** [comment]
Yup! It's iojs' breaking change. Found it [here](https://github.com/nodejs/node/pull/278). I believe iojs is doing it right. I'm trying to see if using URL package [here](https://github.com/defunctzombie/node-url) fixes the issue in all VMs?

**Comment by IvanGoncharov** [comment]
IMHO, its better to use https://github.com/medialize/URI.js 
It simplify a code and remove workarounds like [this](https://github.com/apigee-127/swagger-converter/blob/master/index.js#L293).
I can make PR for this.

**Comment by mohsen1** [comment]
ok @IvanGoncharov that makes sense. Copying over code is never a good idea. I'll merge this. Please make your changes in master after this merge.

**Pull request merged**